lib/chewy/rake_helper.rb

```diff
@@ -228,6 +232,25 @@ def update_mapping(name:, output: $stdout)
         end
       end
 
+      # Parses options that are required to run journal cleanup asynchronously
+      # @see https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html
+      #
+      # @example
+      #   Chewy::RakeHelper.parse_delete_by_query_options(["chewy:journal:clean", "--", "--wait_for_completion=false", "--requests_per_second=10", "--scroll_size=5000"])
+      #
+      def parse_delete_by_query_options(args)
+        str_to_bool = ->(str) { {'true' => true, 'false' => false}.fetch(str) { raise('unknown value for boolean is provided') } }
+        options = {}
+        opts = OptionParser.new
+        opts.on('-w', '--wait_for_completion=ARG', String) { |val| options[:wait_for_completion] = str_to_bool.call(val) }
```

Comment:
With such option, you don't need string parser.
```suggestion
        opts.on('-w', '--[no-]wait') { |val| options[:wait_for_completion] = val }
```